necroHeal.lic

```diff
@@ -0,0 +1,43 @@
+=begin
+
+  Documentation: https://elanthipedia.play.net/Lich_script_development#necroheal
+  ## ** Placement comments here until finalized and added to pedia.
+
+  Necromancer stand-alone healing script
+  
+  Uses all settings defined in HEAL YAML FILE (default)
+  Consume logic via combat trainer
+  
+  TO DO:
+  - See pull request #1462 for harvested materials.
+  - See if there is a background way to use common-healing to determine wound status

```

Comment:
Ahhhh. That's right, safe-room gives a wound status level.  That could be interesting.  Are you thinking I would connect it the safe-room script?

So part of the plan here was to make something useable on the fly, so if you are running around areas exploring, pvping, general mayhem, you could slam necroheal, run out to goblins and heal up real quick.

It's also been turning into a nice integration as an after: script if i would rather heal up in a specific area versus when hunting.  The safe-room would be a nice option for this purpose but sort of defeats the stand alone aspect.   What do you think?